style.css

```diff
@@ -0,0 +1,130 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 10px 8px;
+  background: transparent;
+  font-family: sans-serif;
+  background-color: #f7f7f9;
+  background-image: url(images/Headerbg.png);
+}
```

Comment:
- [ ] You are almost there! However, your whole `body` is using `sans-serif` for texts, let's use Poppins as shown in the Figma template.

- [ ] Nice job using CSS to set your section's background, but you are using the desktop version background which is different from the mobile version background, you can find it on the left sidebar on the Figma app.

Desktop bg:
![Captura de pantalla 2022-04-23 a la(s) 12 08 17 a  m](https://user-images.githubusercontent.com/83717285/164878641-88fcaa29-2b04-49d0-8df7-a0bf41fcb0e2.png)

Mobile bg:
![Captura de pantalla 2022-04-23 a la(s) 12 08 32 a  m](https://user-images.githubusercontent.com/83717285/164878647-d3d2ee0e-4312-41f7-8b3d-6fcccafb802e.png)

Reply:
Okay now I am using Poppins.